packages/graphback-core/src/db/defaultNameTransforms.ts

```diff
@@ -1,19 +1,30 @@
-import { NameTransformDirection } from '../abstract/generateAbstractDatabase'
+export type NameTransformDirection = 'from-db' | 'to-db'
+
+export type NameTransform = (
```

Comment:
Generic name, No comments. Hard to get sense of what this does

Reply:
Agreed, I moved this from `graphql-migrations`.